### DISCUSSION TITLE
Unable to login with custom jwt provider

### DISCUSSION THREAD

**Discussion by zolottareva** [Web3Auth/Web3Auth#1353] [Help]
When trying to login via custom jwt provider (AWS Cognito) a safari page with web3auth open login opens, loads and stucks without redirecting through deep link. 
Any help would be appreciated, been stuck with this for a while
Code snippet:
```
        const web3auth = new Web3Auth(WebBrowser, {
            clientId: clientId,
            network: OPENLOGIN_NETWORK.TESTNET,
            redirectUrl: deepLink,
            loginConfig: {
              jwt: {
                verifier,
                typeOfLogin: LOGIN_PROVIDER.JWT
              },
            }
        });
        const data = await web3auth.login({
            loginProvider: LOGIN_PROVIDER.JWT,
            extraLoginOptions: {
                id_token: token,
                verifierIdField: "username"
            },
        });
```
<img width="313" alt="Screenshot 2023-02-20 at 10 31 36 PM" src="https://user-images.githubusercontent.com/52110829/220193868-5991b9d4-2e74-4c64-b54f-fc56829f2f26.png">

**Comment by EnettyTech** [comment]
Same issue, any solusion?
